.gitattributes

```diff
@@ -39,6 +41,8 @@ ocamlbuild/TODO          ocaml-typo=missing-header
 ocamldoc/Changes.txt     ocaml-typo=missing-header
 ocamldoc/ocamldoc.sty    ocaml-typo=missing-header
 
+patches/Patches.md       ocaml-typo=very-long-line,long-line,missing-header
```

Comment:
> Line breaks are respected in Markdown (at least that seemed to be what was happening?)

No, markdown follows the usual paragraphs are delimited by an empty line. Line breaks are only respected if you end a line with two spaces.